pkg/util/goroutinemap/goroutinemap_test.go

```diff
@@ -39,11 +39,14 @@ const (
 	// wait for an operation to complete (each successive failure results in
 	// exponential backoff).
 	initialOperationWaitTimeLong time.Duration = 500 * time.Millisecond
+
+	// volumeOperationMaxBackoff is the max backOff time of volume operations
+	volumeOperationMaxBackoff time.Duration = 30 * time.Second
 )
 
 func Test_NewGoRoutineMap_Positive_SingleOp(t *testing.T) {
 	// Arrange
-	grm := NewGoRoutineMap(false /* exponentialBackOffOnError */)
+	grm := NewGoRoutineMap(false, volumeOperationMaxBackoff)
```

Comment:
nit: leave the parameter comment (` /* exponentialBackOffOnError */`) in place. Otherwise it makes it difficult for a reader to know what `false` or `true` correspond to.